### PR TITLE
Added Diplomacy Game

### PR DESCRIPTION
**Pull request by chinmay7016**
#269 

Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

**Comment by chinmay7016** [comment]
@edm00se if you like this changes pls merge this pr

File: readme.md [hunk 1]
```diff
@@ -2522,3 +2522,17 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 Contributions are welcome and encouraged! Read the [contribution guidelines][contrib-guide] first.
 
 [contrib-guide]: contributing.md
+
+### [Diplomacy](https://boardgamegeek.com/boardgame/483/diplomacy) 
+
+> This classic game of pure negotiation has taken many forms over the years.
+
+The first The Avalon Hill Game Co version has perhaps the widest release, but Avalon Hill re-released the game in 1999, complete with a colorful new map and metal pieces. In 2008, Avalon Hill released a 50th anniversary edition with a new map and cardboard pieces representing the armies and navies.
```
Comment (edm00se) [inline readme.md:2530]: Please format the rest of the description block, they’ll need the leading `> ` for each line.

File: readme.md [hunk 2]
```diff
@@ -2522,3 +2522,17 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
 Contributions are welcome and encouraged! Read the [contribution guidelines][contrib-guide] first.
 
 [contrib-guide]: contributing.md
+
```
Comment (edm00se) [inline readme.md:2525]: If you look up a line or two, you can see the addition went too low, below the “contribute” paragraph. Please move your addition above this, preferably to one of the sections that makes sense, like “strategy”.

**Review by edm00se — CHANGES_REQUESTED** [review]
The content changes look good, but to get this merged we’ll need:
- an update to the description block markdown formatting
- to move the addition above the “contribute” section, preferably to the strategy section of the document

**Review by chinmay7016 — COMMENTED** [review]
sir pls see the updated file

**Comment by chinmay7016** [comment]
@edm00se  sir pls review changes and merge it pls sir.

File: readme.md (hunk 1 above)
Comment (edm00se) [reply readme.md]: I see the entry got moved, the only thing I see now is that both other paragraphs for the description needs to be formatted with the leading `>`.

**Comment by edm00se** [comment]
I see the entry got moved, the only thing I see now is that both other paragraphs for the description needs to be formatted with the leading `>`.

**Comment by chinmay7016** [comment]
@edm00se sir pls review changes and merge it pls sir.

**Review by edm00se — APPROVED** [review]

**Pull request merged**